api/apps/articles/model/save.coffee

```diff
@@ -136,7 +136,6 @@ sanitize = (article) ->
   if article.sections
     sections = for section in article.sections
       section.body = sanitizeHtml section.body if section.type is 'text'
-      section.caption = sanitizeHtml section.caption if section.type is 'image'
       section.url = sanitizeLink section.url if section.type is 'video'
```

Comment:
This reminds me, we should do another audit of new fields (postscript comes to mind) that could be an XSS risk.

Reply:
Yes for sure!